packages/vue/src/components/organisms/SfHeader/SfHeader.stories.js

```diff
@@ -1,7 +1,18 @@
-import { SfHeader } from "@storefront-ui/vue";
+import {
```

Comment:
Why are we using lots of other components for showcase of sfHeader? what is the purpose of WithHeaderNavigation story?

Reply:
it's very confusing this way for user to play around with the component itself. You can add this as a separate story under Example or an advanced story in SfHeader, but I don't want to mess up the Common case, which is supposed to be the really basic.